requirements.txt

```diff
@@ -1 +1 @@
-agpypeline==0.0.20
\ No newline at end of file
+agpypeline==0.0.39
```

Comment:
This is not the latest out there & there isn't a tag in the GitHub repo for this version. It needs to either be something that's already tagged in GitHub or that is planned to be tagged (preferably the prior). Please change